spec/lib/utils/job-utils/redfish-tool-spec.js

```diff
@@ -0,0 +1,64 @@
+// Copyright 2016, EMC, Inc.
+/* jshint node: true */
+
+'use strict';
+
+describe("redfish-tool", function() {
+    var redfishTool;
+    
+    before(function() {
+         helper.setupInjector([
+            helper.require('/lib/utils/job-utils/redfish-tool.js'),
+        ]);
+        redfishTool = helper.injector.get('JobUtils.RedfishTool');
+    });
```

Comment:
I think you forgot to backup the `process.env.NODE_TLS_REJECT_UNAUTHORIZED` before starting this test suite and recovery it after test suite done. We cannot ensure mocha runs the test cases in a sequence just like we wrote. So it's better to do the backup and recovery global variables in order to not impact other test suites.